src/main/java/application/rest/v1/actions/GlobalResolver.java

```diff
@@ -25,13 +25,13 @@ public String getName() {
     }
     
     @Override
-    public String resolve(ResolutionContext context, String suffix) {
+    public String resolve(ResolutionContext context, String suffix) throws PatternException {
         int i = suffix.indexOf('#');
         if (i >= 0) {
             final String mapName = suffix.substring(0, i);
             final String mapField = suffix.substring(i + 1);
             return context.getConfigMapDataField(mapName, mapField);
         }
-        return null;
+        throw new PatternException("Can not resolve " + suffix + " because it does not contains #");
```

Comment:
Please update this message to say "Cannot resolve " + suffix + " because syntax is incorrect:  expected '#' not found."